src/migtd/Cargo.toml

```diff
@@ -60,3 +60,5 @@ vmcall-raw = ["vmcall_raw"]
 test_heap_size = ["td-benchmark", "td-payload/test_heap_size"]
 test_stack_size = ["td-benchmark"]
 test_disable_ra_and_accept_all = ["attestation/test"] # Dangerous: can only be used for test purpose to bypass the remote attestation
+test_force_success_status = [] # Force migration status to always report success regardless of actual result
+test_force_unsupported_status = [] # Force migration status to always report unsupported regardless of actual result
```

Comment:
Good idea to add feature flags. With the feature flags, it make sense to modify session.rs code to only invoke the "approve-all" code flow, and keep the existing code logic of RATLS. It will be easier to maintain the "approve-all" build when we merge in upstream update.  Suggest to rename the feature flags as "approve-all" and "reject-all".